Tools/LyTestTools/tests/unit/test_file_system.py

```diff
@@ -59,6 +59,11 @@ def test_CheckFreeSpace_EnoughSpace_NoRaise(self, mock_disk_usage):
 
 class TestSafeMakedirs(unittest.TestCase):
 
+    @mock.patch('os.makedirs')
+    def test_SafeMakedirs_ValidCreation_Success(self, mock_makedirs):
+        file_system.safe_makedirs('dummy')
```

Comment:
This function hides most of its side effects, so this is a reasonable sanity test despite there being no test assertion. Though I notice the function's docstring/comments lies about how it emits errors.